inc/template.php

```diff
@@ -47,16 +57,27 @@ function get_author_ids( WP_Post $post ) : array {
  */
 function get_authors( WP_Post $post ) : array {
 	$author_ids = get_author_ids( $post );
+
 	if ( empty( $author_ids ) ) {
 		return [];
 	}
 
-	/** @var WP_User[] */
+	// Just a very simple way to convert author IDs into a string that can be used as a cache key.
+	// Could also generate a hash such as hash( 'crc32', json_encode( $author_ids ) ); .
```

Comment:
My apologies - I think you can just remove this comment completely. I added them mostly as a suggestion to you, it's ultimately your call whether just `implode` is sufficient or whether to use a hashing function. But either way you need to clean up this